### PR TITLE
Verify the no-trunc option for the search operation

### PR DESCRIPTION
**Pull request by sunyuan3**
Signed-off-by: Yuan Sun sunyuan3@huawei.com

### CODE REVIEW

File: integration-cli/docker_cli_search_test.go [hunk 1]
```diff
@@ -63,6 +63,16 @@ func (s *DockerSuite) TestSearchCmdOptions(c *check.C) {
 		c.Fatalf("failed to search on the central registry: %s, %v", outSearchCmd, err)
 	}
 
+	searchCmdNotrunc := exec.Command(dockerBinary, "search", "--no-trunc=true", "busybox")
+	outSearchCmdNotrunc, exitCode, err := runCommandWithOutput(searchCmdNotrunc)
+	if err != nil || exitCode != 0 {
```
Comment (runcom) [inline integration-cli/docker_cli_search_test.go]: no big deal but exitCode should be always not equal to zero if an error occurred

Comment (tiborvass) [inline integration-cli/docker_cli_search_test.go]: you can remove exitCode check, and just do `outSearchCmdNotrunc, _, err := `

**Comment by sunyuan3** [comment]
ping @tiborvass and @runcom , the codes have been updated. Could you please review them?

**Comment by runcom** [comment]
LGTM (you could even remove `=true` in `--no-trunc=true` flag but just nits of mine lol)

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**